functions/src/access.js

```diff
@@ -1,3 +1,5 @@
+// TODO: Rename this file to access.ts (or access/index.ts) + refactor to make proper use of TypeScript
```

Comment:
[TODO] Rename this file to `access.ts` (or `access/index.ts`) + refactor to make proper use of TypeScript